kasa/cli.py

```diff
@@ -49,10 +54,19 @@ def wrapper(message=None, *args, **kwargs):
 # --json has set it to _nop_echo
 echo = _do_echo
 
-DEVICE_TYPES = [
```

Comment:
This breaks the cli for tapos, but I think it's fine. These will likely be the "types" that will at later point define the common API when we extract interfaces out from the implementations.

Reply:
Agreed